ownerdelete.user.js

```diff
@@ -163,6 +170,22 @@ dry.once("load", () => {
     }
     return file;
   };
+  async function getInfo(file) {
+    try {
+      if (file.id in checksums && !file.checksum) {
+        file.checksum = checksums[file.id];
+        return;
+      }
+      const info = await dry.exts.info.getFileInfo(file.id);
```

Comment:
Really, this line is basically the only line you actually want to run in the promise pool. Well, and some error handling maybe. That way you can avoid going through the microtask queue when the value is already cached.

Actually, I ran into a bug in getFileInfo in volanail that might happen here too: if a file vanishes between queuing on the pool and the pool actually running getFileInfo, then the promise getFileInfo may never resolve, which in turn would block the promise pool too, of course. (lain cannot code).
I didn't address this in volanail yet, tho, as it didn't seem worth spending time on. Basically, a potential fix would be to make a promise that gets resolved by getFileInfo but rejected after a timeout.